.github/workflows/build.yml

```
@@ -0,0 +1,29 @@
+name: Test Scratch Asset Types 
+on:
+    workflow_dispatch: 
+    push:
+jobs:
+    build:
+        runs-on: ubuntu-latest
+        env:
+            NODE_ENV: production
+        steps:
+            - uses: actions/checkout@v4
+            - uses: actions/setup-node@v3
+              with:
+                cache: "npm"
+                node-version-file: '.nvmrc'
+            - name: Run NPM Commands
+              run:  |
+                npm --production=false install
+                npm --production=false update
+                npm --production=false prune
```

Since this has a `package-lock.json` now, I think this should switch to `npm ci`:
```suggestion
                npm --production=false ci
```

---

good call!